dali/operators/reader/loader/video/frames_decoder_gpu.h

```diff
@@ -78,7 +78,7 @@ class NVDECLease {
     }
 
     ~NVDECLease() {
-      Return();
+      Return();  // Intentionally throwing in noexcept scope.
```

Comment:
nitpick: This comment here is a bit misleading (I don't see that we are throwing here). Also, I think that we have this widely accepted (allow exceptions to arise on dtors) that we don't need to add those comments

Reply:
:+1: This comment belongs in coverity report, not in the code.